.github/workflows/release.yml

```diff
@@ -27,6 +27,12 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: 1.17
+    - name: Setup Snapcraft
```

Comment:
I don't think it will be possible. This part makes available the snapcraft command. And it need to be executed before goreleaser. The other part publishes the snaps that already had builded.